src/NuGet.Core/NuGet.Packaging/NuGet.Packaging.csproj

```diff
@@ -49,6 +49,11 @@
   </ItemGroup>
   
   <ItemGroup>
+    <Compile Update="PackageCreation\Resources\NuGetResources.Designer.cs">
```

Comment:
what are these changes for?

Reply:
That was redone by VS when setting the ResX to regenerate the strongly typed files. I added a resource for an exception